rules/0220-msauth_rules.xml

```diff
@@ -922,13 +922,6 @@
     <group>pci_dss_10.2.4,pci_dss_10.2.5,pci_dss_11.4,gdpr_IV_35.7.d,gdpr_IV_32.2,hipaa_164.312.b,nist_800_53_AU.14,nist_800_53_AC.7,nist_800_53_SI.4,</group>
   </rule>
 
-  <rule id="18152" level="10" frequency="$MS_FREQ" timeframe="240">
-    <if_matched_group>win_authentication_failed</if_matched_group>
-    <same_source_ip />
-    <description>Multiple Windows Logon Failures.</description>
-    <group>authentication_failures,pci_dss_10.2.4,pci_dss_10.2.5,pci_dss_11.4,gdpr_IV_35.7.d,gdpr_IV_32.2,hipaa_164.312.b,nist_800_53_AU.14,nist_800_53_AC.7,nist_800_53_SI.4,</group>
-  </rule>
-
   <rule id="18153" level="10" frequency="$MS_FREQ" timeframe="240">
```

Comment:
That's true. 

Neither this rule or rule `40111` have children in the default ruleset, but they may exist in custom rulesets, so it may be best to keep both.

It would probably be best to update its description as there isn't an easy way to discriminate Windows and non-windows system.

I also see now that rule `51004` from `0305-dropbear_rules.xml` is identical and claims the same criteria to match dropbear attacks. 

Would un-removing them and changing their descriptions satisfy the requested change?

Reply:
I agree, that should be the best approach in order to avoid the redundant rules and make it meaningful.